Doc change. Added some system queries (1st attempt). (#6857)
ClickHouse/ClickHouse: docs/en/query_language/system.md

# SYSTEM Queries {#query_language-system}

- [RELOAD DICTIONARIES](#query_language-system-reload-dictionaries)
- [RELOAD DICTIONARY](#query_language-system-reload-dictionary)
- [DROP DNS CACHE](#query_language-system-drop-dns-cache)
- [DROP MARKS CACHE](#query_language-system-drop-marks-cache)
- [FLUSH LOGS](#query_language-system-flush_logs)
- [RELOAD CONFIG](#query_language-system-reload-config)
- [SHUTDOWN](#query_language-system-shutdown)
- [KILL](#query_language-system-kill)
- [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
- [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
- [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)

## RELOAD DICTIONARIES {#query_language-system-reload-dictionaries}

> Reloads all dictionaries that have been successfully downloaded before.

Reviewer: ```suggestion
Reloads all dictionaries that have been successfully loaded before.
```

> By default, lazy loading is enabled [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load), so dictionaries are not loaded automatically at startup, but only when they first access through dictGet or SELECT to ENGINE = Dictionary. `system reload dictionaries` command reloads such dictionaries (LOADED).

Reviewer: ```suggestion
By default, dictionaries are loaded lazily (see [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load)), so instead of being loaded automatically at startup, they are initialized on first access through dictGet function or SELECT to ENGINE = Dictionary. The `SYSTEM RELOAD DICTIONARIES` query reloads such dictionaries (LOADED).
```

> By default, dictionaries are loaded lazily (see [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load)), so instead of being loaded automatically at startup, they are initialized on first access through dictGet function or SELECT to ENGINE = Dictionary. The `SYSTEM RELOAD DICTIONARIES` query reloads such dictionaries (LOADED).

Reviewer: ```suggestion
By default, dictionaries are loaded lazily (see [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load)), so instead of being loaded automatically at startup, they are initialized on first access through dictGet function or SELECT from tables with ENGINE = Dictionary. The `SYSTEM RELOAD DICTIONARIES` query reloads such dictionaries (LOADED).
```

Always returns `Ok.` regardless of the result of the dictionary update.

## RELOAD DICTIONARY dictionary_name {#query_language-system-reload-dictionary}

Completely reloads a dictionary `dictionary_name`, regardless of the state of the dictionary (LOADED / NOT_LOADED / FAILED).
Always returns `Ok.` regardless of the result of updating the dictionary.

> The status of the dictionary can be checked by querying `system.dictionaries`.

Reviewer: ```suggestion
The status of the dictionary can be checked by querying the `system.dictionaries` table.
```

```sql

> select name,status from system.dictionaries

Reviewer: ```suggestion
SELECT name, status FROM system.dictionaries;
```

```

## DROP DNS CACHE {#query_language-system-drop-dns-cache}

Resets ClickHouse's internal DNS cache. Sometimes it is necessary to use this command when changing the infrastructure (changing the IP address of another ClickHouse server or the server used by dictionaries).

> For more convenient (automatic) cache management, see disable_internal_dns_cache, dns_cache_update_period parameters.

Reviewer: links?

Author: They are not documented yet. It's sort of todo for myself.

## DROP MARKS CACHE {#query_language-system-drop-marks-cache}

Resets the mark cache. Used in development of ClickHouse and performance tests.

## FLUSH LOGS {#query_language-system-flush_logs}

Flushes buffers of log messages to system tables (e.g. system.query_log). Allows you to not wait 7.5 seconds when debugging.

## RELOAD CONFIG {#query_language-system-reload-config}

> Reloads ClickHouse configuration. Used when configuration is stored in zookeeeper.

Reviewer: ```suggestion
Reloads ClickHouse configuration. Used when configuration is stored in ZooKeeeper.
```
